README.md

```diff
@@ -4,19 +4,26 @@
 A python client for [senat.fr](https://senat.fr) website.
 
 ## Main goal
+
 Retrieve painlessly JSON data from [senat.fr](https://senat.fr).
 
 ## Requirements
+
 Python 3
 
-## Install :
-```bash
-pip3 install senapy (or "pip3 install -e ." locally)
-```
+## Install:
+
+- From pip: `pip3 install senapy`
+- Locally: `pip3 install -r requirements.txt`
```

Comment:
locally you should use `pip3 install -e .` since `pip install -r` doesn't install `senapy` and it's scripts like `senapy-cli`